lib/post_creator.rb

```diff
@@ -159,7 +159,7 @@ def valid?
         return false
       end
 
-      if @topic&.slow_mode_seconds.to_i > 0
+      if @topic&.slow_mode_seconds.to_i > 0 && !@user.staff?
```

Comment:
Maybe these checks (there is one in post_revisor too) should be in the guardian?

Reply:
Good idea, thanks.